base/channels.jl

```diff
@@ -2,6 +2,56 @@
 
 abstract type AbstractChannel end
 
+if JULIA_PARTR
```

Comment:
should this be `@static if`?

Reply:
they're equivalent at the top-level

Reply:
Good to know, thanks.
There are a few places further down that are not at top-level that should be `@static if` then,
I've now marked them